config/config.html

```diff
@@ -0,0 +1,27 @@
+<div id="template" class="well">
+    <h2>Github Status</h2>
+    <br/>
+    <br/>
+    <h3 class="help-text">
+        Configure context for status messages
+    </h3>
+    <div>context :</div>
+    <input type="text" class="span12" ui-refresh="selectedTab" ng-model="config.context" />
+    <br/>
+    <br/>
+    <h3 class="help-text">
+        Configure description for each status message
+    </h3>
+    <div>pending :</div>
```

Comment:
See the config here https://github.com/Strider-CD/strider-node/blob/master/config/config.html